src/app/helpers.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Determines if variable is null or undefined
+ * @param variable Variable to validate
+ */
+export function isUninitialized(variable: any): boolean {
```

Comment:
this seems a little redundant.

Further I'd argue it is semantically wrong.

What is the point of having both `null` and `undefined`?

I consistently use `undefined` to represent a value that has not been set yet, and `null` to indicate a value that has been set to nothing.

Thus this definition of `isUninitialized` (semantically meaning _'not set yet'_) would break that definition